src/homepageExperience/containers/CliWizard.tsx

```diff
@@ -0,0 +1,222 @@
+
+// Steps
+import {ExecuteAggregateQuery} from 'src/homepageExperience/components/steps/cli/ExecuteAggregateQuery'
+import {ExecuteQuery} from 'src/homepageExperience/components/steps/cli/ExecuteQuery'
+import {Finish} from 'src/homepageExperience/components/steps/Finish'
+import {InitializeClient} from 'src/homepageExperience/components/steps/cli/InitializeClient'
+import {InstallDependencies} from 'src/homepageExperience/components/steps/cli/InstallDependencies'
+import {Overview} from 'src/homepageExperience/components/steps/Overview'
+import {WriteData} from 'src/homepageExperience/components/steps/cli/WriteData'
+import WriteDataDetailsContextProvider from 'src/writeData/components/WriteDataDetailsContext'
+
+// Utils
+import {event} from 'src/cloud/utils/reporting'
+import {HOMEPAGE_NAVIGATION_STEPS_CLI} from 'src/homepageExperience/utils'
+import RateLimitAlert from 'src/cloud/components/RateLimitAlert'
+
+interface State {
+  currentStep: number
+  selectedBucket: string
+  finishStepCompleted: boolean
+  tokenValue: string
+  finalFeedback: number
+}
+
+export class CliWizard extends PureComponent<{}, State> {
+  state = {
+    currentStep: 1,
+    selectedBucket: 'sample-bucket',
+    finishStepCompleted: false,
+    tokenValue: null,
+    finalFeedback: null,
+  }
+
+  private handleSelectBucket = (bucketName: string) => {
+    this.setState({selectedBucket: bucketName})
+  }
+
+  private handleMarkStepAsCompleted = () => {
+    this.setState({finishStepCompleted: true})
+  }
+
+  private setTokenValue = (tokenValue: string) => {
+    this.setState({tokenValue: tokenValue})
+  }
+
+  private setFinalFeedback = (feedbackValue: number) => {
+    this.setState({finalFeedback: feedbackValue})
+  }
+
+  handleNextClick = () => {
+    this.setState(
+      {
+        currentStep: Math.min(
+          this.state.currentStep + 1,
+          HOMEPAGE_NAVIGATION_STEPS_CLI.length
+        ),
+      },
+      () => {
+        event(
+          'firstMile.cliWizard.next.clicked',
+          {},
+          {
+            clickedButtonAtStep: this.state.currentStep - 1,
+            currentStep: this.state.currentStep,
+          }
+        )
+      }
+    )
+  }
+
+  handlePreviousClick = () => {
+    this.setState(
+      {currentStep: Math.max(this.state.currentStep - 1, 1)},
+      () => {
+        event(
+          'firstMile.cliWizard.previous.clicked',
+          {},
+          {
+            clickedButtonAtStep: this.state.currentStep + 1,
+            currentStep: this.state.currentStep,
+          }
+        )
+      }
+    )
+  }
+
+  handleNavClick = (clickedStep: number) => {
+    this.setState({currentStep: clickedStep})
+  }
+
+  renderStep = () => {
+    switch (this.state.currentStep) {
+      case 1: {
+        return <Overview wizard="cliWizard" />
+      }
+      case 2: {
+        return <InstallDependencies />
+      }
+      case 3: {
+        return (
+          <InitializeClient
+            wizardEventName="cliWizard"
```

Comment:
it doesn't seem like the property `wizardEventName` is necessary for the CLI onboarding, since it's a completely separate experience that doesn't share its components with any other experience (unlike the python, node, and go experiences which share a bit with each other). 

I wrote up a ticket to handle this: https://github.com/influxdata/ui/issues/5177